sigtypes.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package winmd
+
+import (
+	"github.com/microsoft/go-winmd/flags"
+)
+
+type SigType uint8
```

Comment:
This type seems to be unused.

Reply:
Yeah, didn't end up with a "read any signature blob" API where this would have been set by the parser and used by the caller to distinguish types of signature. I think this is generally a good thing. I'm hoping I'm not missing a situation where some truly arbitrary signatures are floating around with no context. Can always add this back in if needed.